docs/src/pages/de/migration/0.21.0.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Umstellung auf v0.21
+description: Projekte von Astro v0.20 umziehen.
+---
+
+## Vite
+
+Beginnend mit v0.21 werden Astro-Builds mittels [Vite] kompiliert.
```

Comment:
You forgot to link to <https://vitejs.dev/>

Reply:
Done.